dbase.py

```diff
@@ -0,0 +1,19 @@
+import mysql.Connector
+
+
+dbl=mysql.connector.connect(host='localhost', user='root', password='1234', database='user')
+
+cur=db1.cursor()
+
+r=int(input("Enter your rollno\n"))
+n=input("Enter your name\n")
+m=float(input("Enter registration number"))
+
+
+s='INSERT INTO details values(%s,%s,%s)'

```

Comment:
Refer [This](https://www.geeksforgeeks.org/django-orm-inserting-updating-deleting-data/) and [official documentation](https://docs.djangoproject.com/en/4.0/topics/db/queries/)

Reply:
Okay